practice/react-practice-one/src/components/Select/Select.jsx

```diff
@@ -0,0 +1,53 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import CSS */
+import './select.css';
+
+/* Import components */
+import SelectIcon from '../Icons/SelectIcon';
+
+const Select = ({ options, onChange }) => {
+  return (
+    <div className="select-overlay ">
+      <SelectIcon />
+      <select className="sort-container" onChange={onChange}>
+        <option value="default">Sort by</option>
```

Comment:
How can we use this component in other cases?

Reply:
I fixed it